src/ImageManipulation.php

```diff
@@ -581,20 +581,25 @@ public function getIcon()
     }
 
     /**
+     * Cache of image backends
+     *
+     * @var array
+     */
+    protected static $image_backend_cache = [];
```

Comment:
We should be using non-config statics extremely sparingly in any case. We should use psr-2.